TACC/Core-Portal
# bug/FP-655: Fix filtering of interactive session notifications causing "Connect" button to not show

## Overview: ##
In `History.js`, an `eventType` filter was preventing `interactive_session_ready` events from propagating to the Job History view. This fixes that by including both event_types in the filter on the backend.

## PR Status: ##


## Related Jira tickets: ##

* [FP-655](https://jira.tacc.utexas.edu/browse/FP-655)

## Summary of Changes: ##

## Testing Steps: ##
1. Launch an interactive job, like RStudio
2. Watch "Session Ready" event toast come in, and see "Connect" button appear in History -> Jobs view
3. Confirm "Connect" button persists on refresh as long as the session is still running

## UI Photos:

## Notes: ##
